srcs/app/http/HttpRequest.cpp

```diff
@@ -1,8 +1,12 @@
 #include <HttpRequest.hpp>
+#include <ostream>
 #include <stdexcept>
+#include <cerrno>
```

Comment:
Al ver esto me ha dado yuyu por lo de la advertencia que tenemos en el enunciado:
"Checking the value of errno is strictly forbidden after a read or a write operation"
Pero lo usas solo para ver que strtoul no se salga de rango. Me parece bien. Yo también uso esa funcion cuando puedo.

Reply:
Es conveniente en algunos casos, y sí, hay que tener cuidado con ello :sweat_smile: